File: TCPServer/Javascript/TCPClient.js
const net=require("net")
const stdin=process.openStdin()   //Get stdin handle
stdin.addListener("data",message=>
{
 const client=new net.Socket()    //Once something is read from stdin create a new socket
 client.connect(3721,"127.0.0.1") //Connect to server
 client.write(message)            //Write the message to the server
 client.on('data',data=>          //When a reply is recieved write it to stdout
 {
  process.stdout.write(data.toString('ascii'))
 })
 client.close()                   //Close the client
})

File: TCPServer/Python/TCPClient.py
import socket
while True:
 message=input()                                         #Read message from console
 client=socket.socket(socket.AF_INET,socket.SOCK_STREAM) #Create client socket
 client.connect(("127.0.0.1",3721))                      #Connect to server
 client.send(bytes(message+"\n","ascii"))                #Send message
 reply=client.recv(0xFF)                                 #Recieve reply from server
 print(reply.decode("ascii"),end='')                     #Print reply to console
 client.close()                                          #Close client